HomeTask5/HomeTask5/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace HomeTask5
+{
+    class Program
+    {
+        static void Main()
+        {
+            string StringKosmos, StringSmeh;
+            string FirstString = "микросхема";
```

Comment:
Локальные переменные принято называть маленькой буквы
```suggestion
            string stringKosmos, stringSmeh;
            string firstString = "микросхема";
```